.stylelintrc.json

```diff
@@ -0,0 +1,32 @@
+{
+    "extends": ["stylelint-config-standard"],
+    "plugins": ["stylelint-scss", "stylelint-csstree-validator"],
+    "rules": {
+      "at-rule-no-unknown": [
+        true,
+        {
+          "ignoreAtRules": [
+            "tailwind",
+            "apply",
+            "variants",
+            "responsive",
+            "screen"
+          ]
+        }
+      ],
+      "scss/at-rule-no-unknown": [
+        true,
+        {
+          "ignoreAtRules": [
+            "tailwind",
+            "apply",
+            "variants",
+            "responsive",
+            "screen"
+          ]
+        }
```

Comment:
![image](https://user-images.githubusercontent.com/93039359/213360242-af2112c8-063e-4bf2-a839-a4daa5d9dc90.png)

- [ ] You have not made any models in your application :fearful: You are not following project requirements in your application :thinking: Kindly make models using the ER Diagram given you in the project requirements :+1: